config/locales/en.yml

```diff
@@ -194,8 +304,14 @@ en:
     open_source_statement_1: 'Gameboard and registration software is '
     open_source_statement_2: ' free to use!'
     github_link: 'https://github.com/mitre-cyber-academy/ctf-scoreboard'
```

Comment:
I don't think so, It's like that because there is a hyperlink between these two lines.

Reply:
The github_link can be its own line, however I think the open source statement 1 and 2 can be combined like so: https://stackoverflow.com/questions/2543936/rails-i18n-translating-text-with-links-inside

Reply:
It should work that easily, however because rails encapsulates all elements in quotes, not so much. 
![Annotation 2020-06-18 100847](https://user-images.githubusercontent.com/66680985/85030658-b96fd900-b14b-11ea-994e-4427829cd071.png)

There is a gem that was linked in that SO question which does work, but adds yet another dependency. I'll make a commit using it, though.